pyproject.toml

```diff
@@ -16,6 +17,7 @@ dependencies = [
     'numpy',
     'scipy',
     'matplotlib',
+    'torch',
```

Comment:
At some point we might want to make this optional, for users that only use the linear stuff.

Reply:
We can do this now but I didn't have the code to gracefully warn and exit if networks is imported without the optional dependency to hand. If this is easy and I'm just unaware then go ahead with this

Reply:
Let's do it at a later date